nap/docker/ckan/ckan-plugins/ckanext-napote_theme/ckanext/napote_theme/templates/header.html

```diff
@@ -70,6 +69,13 @@
 
                         {% block header_account_container_content %}
                             {% if c.userobj %}
+                                <ul class="nav nav-pills">
+                                    <li>
+                                        <a href="https://s3.eu-central-1.amazonaws.com/ote-assets/nap-ohje.pdf" target="_blank" title="NAP-palvelun käyttöohje">
```

Comment:
Translate title

Reply:
I Copy-pasted the old implementation. I'll add the missing translation key.